guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,385 @@
+```
+
+#### Guild#voice
+
+The `Guild#voice` getter has been removed.
+
+```diff
+- guild.voice
++ guild.me.voice
+```
+
+### GuildMember 
+
+#### GuildMember#ban
+
+`GuildMember#ban()` will throw a TypeError when a string is provided instead of an options object.
+
+```diff
+- member.ban('reason')
++ member.ban({ reason: 'reason' })
+```
+
+### GuildMember#hasPermission
+
+This shortcut method has been removed.
+
+```diff
+- member.hasPermission(Permission.FLAGS.SEND_MESSAGES);
++ member.permissions.has(Permission.FLAGS.SEND_MESSAGES);
+```
+
+### GuildMemberManager
+
+#### GuildMemberManager#ban
+
+``GuildMemberManager#ban` will throw a TypeError when a string is provided instead of an options object.
+
+```diff
+- guild.members.ban('123456789012345678', 'reason')
++ guild.members.ban('123456789012345678', { reason: 'reason' })
+```
+
+### Message / MessageManager
+
+#### Message#delete
+
+`Message.delete()` no longer accepts any options, requiring a timed-delete to be performed manually.
+
+```diff
+- message.delete({ timeout: 10000 });
++ client.setTimeout(() => message.delete(), 10000);
+```
+
+`reason` is no longer a parameter as it is not used by the API.
+
+#### MessageManager#delete
+
+`MessageManager.delete()` no longer accepts any additional options, requiring a timed-delete to be performed manually.
+
+```diff
+- channel.messages.delete(''123456789012345678', { timeout: 10000 });
++ client.setTimeout(() => channel.messages.delete('123456789012345678'), 10000);
+```
+
+`reason` is no longer a parameter as it is not used by the API.
+
+
+### RoleManager
+
+#### RoleManager#create
+
+The options passed to `RoleManager#create` no longer need to be nested in a `data` object.
+
+Additionally `reason` is now part of the options, not a second parameter
+
+```diff
+- guild.roles.create({ data: { name: "New role" } }, "Creating new role");
++ guild.roles.create({ name: "New role", reason: "Creating new role" })
+```
+
+#### RoleManager#fetch
+
+`RoleManager.fetch()` will now return a Collection instead of a RoleManager when called without params.
+
+### User
+
+#### User#locale
+
+`User.locale` has been removed, as this property is not exposed to bots.
+
+### UserFlags
+
+The deprecated UserFlags `DISCORD_PARTNER` and `VERIFIED_DEVELOPER` / `EARLY_VERIFIED_DEVELOPER` have been removed in favour of their renamed versions.
+
+```diff
+- user.flags.has(UserFlags.FLAGS.DISCORD_PARTNER)
++ user.flags.has(UserFlags.FLAGS.PARTNERED_SERVER_OWNER)
+
+- user.flags.has(UserFlags.FLAGS.VERIFIED_DEVELOPER)
++ user.flags.has(UserFlags.FLAGS.EARLY_VERIFIED_BOT_DEVELOPER)
+```
+
+### Util
+
+#### Util#convertToBuffer
+
+#### Util#str2ab
+
+Both removed in favour of Node built-in Buffer methods.
+
+## Additions
+
+### ActivityTypes
+
+New activity type `COMPETING` added.
+
+### Channel
+
+#### Channel#isText()
+
+The new `Channel#isText()` getter provides an easy way for TypeScript developers to determine if a channel is Text-Based ("dm", "text", "news")
+
```

Comment:
```suggestion

```
This already exists on v12

Reply:
It's still a new feature, I don't see the harm in listing it